src/easyscore.js

```diff
@@ -271,6 +284,10 @@ export class EasyScore {
     this.options = Object.assign({
       factory: null,
       builder: null,
+      commitHooks: [
+        setId,
```

Comment:
Should this support multiple tokens? And if so, separated by comma or space or...?